reconcile/gitlab_mr_sqs_consumer.py

```diff
@@ -0,0 +1,45 @@
+"""
+SQS Consumer to create Gitlab merge requests.
+"""
+
+import json
+import logging
+
+import reconcile.queries as queries
+
+from utils import mr
+from utils.sqs_gateway import SQSGateway
+from utils.gitlab_api import GitLabApi
+
+
+QONTRACT_INTEGRATION = 'gitlab-mr-sqs-consumer'
+
+
+def run(gitlab_project_id, dry_run):
+    settings = queries.get_app_interface_settings()
+
+    accounts = queries.get_aws_accounts()
+    sqs_cli = SQSGateway(accounts, settings=settings)
+
+    instance = queries.get_gitlab_instance()
+    saas_files = queries.get_saas_files_minimal()
+    gitlab_cli = GitLabApi(instance, project_id=gitlab_project_id,
+                           settings=settings, saas_files=saas_files)
+
+    while True:
+        messages = sqs_cli.receive_messages()
+        logging.info('received %s messages', len(messages))
+
+        if not messages:
+            break
+
+        for message in messages:
+            receipt_handle, body = message[0], message[1]
+
+            logging.info('received message %s with body %s',
+                         receipt_handle[:6], json.dumps(body))
+
+            if not dry_run:
+                merge_request = mr.init_from_sqs_message(body)
+                merge_request.submit_to_gitlab(gitlab_cli=gitlab_cli)
+                sqs_cli.delete_message(receipt_handle)
```

Comment:
can we defer the delete_message? if a message is faulty i think that we currently don't want to keep it in the queue (we'll need to think about an error queue at a later point, but not in scope for this PR).